activetable/templates/html/activetable.html

```diff
@@ -0,0 +1,47 @@
+<div class="activetable_block">
+  {% if help_text %}
+  <div class="activetable-help" style="width: {{ total_width }}px;">
+    <button id="activetable-help-button">+help</button>
+    <p id="activetable-help-text">{{ help_text }}</p>
+  </div>
+  {% endif %}
+  {% if thead %}
+  <table id="activetable">
+    <colgroup>
+      {% for width in column_widths %}<col style="width: {{ width }}px;">{% endfor %}
+    </colgroup>
+    <thead>
+      <tr style="height: {{ head_height }}px;">
+        {% for cell in thead %}<th>{{ cell }}</th>{% endfor %}
+      </tr>
+    </thead>
+    <tbody>
+      {% for row in tbody %}
+      <tr class="{{ row.class }}" style="height: {{ row.height }}px;">
+        {% for cell in row.cells %}
+        <td class="{{ cell.classes }}" id="{{ cell.id }}">
+          {% if cell.is_static %}
+          {{ cell.value }}
+          {% else %}
+          <input type="text" style="height: {{ cell.height }}px;" size=1
```

Comment:
@cptvitamin Is it acceptable to use an `aria-label` attribute on the input instead of a `label` tag?  I don't want to show the label to sighted users, so using `aria-label` seems like the easiest option.